src/theme.ts

```diff
@@ -466,35 +509,137 @@ export const newTheme = createTheme({
         },
       },
     },
+    MuiTypography: {
+      defaultProps: {
+        variantMapping: {
+          bodyMdBold: 'p',
+          bodyMdRegular: 'p',
+          bodySmBold: 'p',
+          bodySmRegular: 'p',
+          headingLg: 'h1',
+          headingMd: 'h2',
+          headingSm: 'h3',
+        },
+      },
+    },
   },
   elevation: {
     bottom: {
       big: {
-        light: '0px 4px 40px 0px #00000014',
-        medium: '0px 4px 40px 0px #0000001F',
+        light: '0rem 0.25rem 2.5rem 0rem #00000014',
+        medium: '0rem 0.25rem 2.5rem 0rem #0000001F',
       },
       small: {
-        light: '0px 4px 20px 0px #00000014',
-        medium: '0px 4px 20px 0px #0000001F',
+        light: '0rem 0.25rem 1.25rem 0rem #00000014',
+        medium: '0rem 0.25rem 1.25rem 0rem #0000001F',
       },
     },
     top: {
       big: {
-        light: '0px -4px 40px 0px #00000014',
-        medium: '0px -4px 40px 0px #0000001F',
+        light: '0.rem -0.25rem 2.25rem 0rem #00000014',
+        medium: '0rem -0.25rem 2.25rem 0rem #0000001F',
       },
       small: {
-        light: '0px -4px 20px 0px #00000014',
-        medium: '0px -4px 20px 0px #0000001F',
+        light: '0rem -0.25rem 1.25rem 0rem #00000014',
+        medium: '0rem -0.25rem 1.25rem 0rem #0000001F',
       },
     },
   },
   palette: newThemePalette,
   typography: {
```

Comment:
I see there is an inconsistency between Figma and what is being displayed in implementation. 

In Figma I can see that for example the Semi-bold font-weight of Figtree corresponds to 600 but when applied in CSS it becomes too thick. 

You can see several examples of this in the images below:

<img width="293" alt="image" src="https://github.com/user-attachments/assets/5a0989d1-c0ea-425e-9883-40ec383c1704">

<img width="216" alt="image" src="https://github.com/user-attachments/assets/f42e5741-2861-4512-a0d3-080d4189aa1a">

<img width="298" alt="image" src="https://github.com/user-attachments/assets/6220c2b5-4fd7-4f6b-98c3-532341b99c6c">

So in order to correct this we need to bring down all font-weights by 100.

- [ ] 600 -> 500
- [ ] 500 -> 400
- [ ] 400 -> 300